cns/restserver/api_test.go

```diff
@@ -1302,6 +1302,31 @@ func TestNmAgentSupportedApisHandler(t *testing.T) {
 	fmt.Printf("nmAgentSupportedApisHandler Responded with %+v\n", nmAgentSupportedApisResponse)
 }
 
+func TestNMAgentNCListHandler(t *testing.T) {
```

Comment:
I think we should enhance this test by passing in the JSON body with NC IDs, then decoding the JSON body successfully and matching the NC ID.

Reply:
Added. I created an NCID with a random guid and mocked NMA's response. After I call CNS' endpoint of the new API, I receive the same NC ID in the NC list.